catalog-rest-service/src/main/resources/ui/src/components/Modals/FormModal/index.test.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+  * Licensed to the Apache Software Foundation (ASF) under one or more
+  * contributor license agreements. See the NOTICE file distributed with
+  * this work for additional information regarding copyright ownership.
+  * The ASF licenses this file to You under the Apache License, Version 2.0
+  * (the "License"); you may not use this file except in compliance with
+  * the License. You may obtain a copy of the License at
+
+  * http://www.apache.org/licenses/LICENSE-2.0
+
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+*/
+
+import {
+  findByTestId,
+  findByText,
+  fireEvent,
+  render,
+} from '@testing-library/react';
+import React from 'react';
+import FormModal from '.';
+
+const mockCancel = jest.fn();
+const mockSave = jest.fn();
+const mockForm = jest.fn().mockReturnValue(<p data-testid="form">data</p>);
```

Comment:
Why do we need `data-testid` here?

Reply:
for checking mocked component is rendered or not.